### PR TITLE
Name your PRs appropriately ree

### PR DESCRIPTION
**Pull request by ACCount12**
This PR renames "station holomap". Because whoever has ported it to CEV "Eris", the codebase named after the goddamn _ship_, has never thought of doing that!

🆑
tweak: Renames station holomap to ship holomap
fix: fixes uranium ore export requiring diamond sheets
/🆑

### CODE REVIEW

File: code/modules/cargo/exports/sheets.dm [hunk 1]
```diff
@@ -145,7 +145,7 @@
 // Uranium. Still useful for both power generation and nuclear annihilation.
 /datum/export/stack/uranium
 	cost = 20
-	export_types = list(/obj/item/stack/material/diamond)
+	export_types = list(/obj/item/stack/material/uranium)
```
Comment (drexample) [inline code/modules/cargo/exports/sheets.dm:148]: unmentioned change

**Comment by Kurgis** [comment]
`#define HOLOMAP_EXTRA_STATIONMAP			"stationmapformatted"`
`#define HOLOMAP_EXTRA_STATIONMAPAREAS		"stationareas"`
`#define HOLOMAP_EXTRA_STATIONMAPSMALL		"stationmapsmall"`

I suppose you better change these defines then.

**Comment by MadmanMartian** [comment]
nuclear option
`#define station ship`

And let the compilers do its work

File: code/modules/cargo/exports/sheets.dm (hunk 1 above)
Comment (ACCount12) [reply code/modules/cargo/exports/sheets.dm:148]: Some random shitty export datum fix got in. No harm done.

**Pull request merged**